crates/ruff_cli/Cargo.toml

```diff
@@ -65,9 +65,6 @@ ureq = { version = "2.6.2", features = [] }
 [features]
 jupyter_notebook = ["ruff/jupyter_notebook"]
 
-[package.metadata.maturin]
-name = "ruff"
```

Comment:
I feel like this was necessary for some reason.

Reply:
yes, but it's now moved to pyproject.toml: https://github.com/PyO3/maturin/blob/main/Changelog.md#01416---2023-03-26

Reply:
Mmm, maybe not?

Reply:
Can confirm, saw this error when building locally with pip too.

Reply:
wdym? the deprecation warning comes from https://github.com/charliermarsh/ruff/actions/runs/4614388337/jobs/8157296671#step:5:531 and i've built the wheel and it looks correct

Reply:
@konstin Referring to this line: https://github.com/charliermarsh/ruff/blob/280dffb5e161acb9deab97c9a2158055e4174d66/pyproject.toml#L2

Reply:
hmm i actually always get the correct wheel with just `maturin build` and no name explicitly set now, but that also seems correct since we're not shipping a python package but only a script which already has the correct name set: https://github.com/charliermarsh/ruff/blob/07759652c68a85efa9eebdff600c34a5d72fa588/crates/ruff_cli/Cargo.toml#L15-L16